scripts/get-members.ts

```diff
@@ -0,0 +1,33 @@
+import { db } from './firebase';
+
+let getNetId = (email: string): string => {
+  let pos = email.search('@');
+  return email.slice(0, pos);
+};
+
+const fs = require('fs');
+const dirPath: string = 'scripts/members/';
+fs.mkdirSync(dirPath, { recursive: true });
+
+const jsonFilesList: string[] = fs.readdirSync(dirPath);
+jsonFilesList.forEach((json) =>
+  fs.unlink(dirPath + json, (err) => {
+    if (err) {
+      console.log(err);
+    }
+  })
+);
+
+db.collection('members')
+  .get()
+  .then((memberRefs) => {
+    memberRefs.forEach((memberRef) => {
+      let member = memberRef.data();
+      let netId = getNetId(member.email);
+      fs.writeFile(dirPath + `${netId}.json`, JSON.stringify(member), (err) => {
```

Comment:
My thought with deleting the existing files in the directory was just that it would keep the members completely up to date with the database when building (in case someone got deleted or something). It was also helpful with testing in case I did anything with the directory. I'm not actually sure if that's completely necessary tbh.

Reply:
With regards to the path, I think that's trivial at the moment (??) since this script ultimately has to run on the nova repo so the path will be entirely different.